### PR TITLE
arm9 MI_init, MI_wram and MI_uncompress + error_handling

### PR DESCRIPTION
**Pull request by red031000**

### CODE REVIEW

**Review by ProjectRevoTPP — DISMISSED** [review]

File: arm9/src/error_handling.c [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+THUMB_FUNC void ErrorHandling(void)
+{
+    if (FUN_02031810())
+    {
+        if (OS_GetProcMode() != OS_PROCMODE_IRQ)
+        {
+            PrintErrorMessageAndReset();
+        }
+    }
+}
```
Comment (PikalaxALT) [inline arm9/src/error_handling.c:16]: Missing newline at EOF

File: arm9/lib/src/MI_init.c [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#include "nitro/types.h"
+#include "function_target.h"
+#include "MI_dma.h"
+#include "MI_wram.h"
+
+ARM_FUNC void MI_Init(void)
+{
+    MI_SetWramBank(MI_WRAM_ARM7_ALL);
```
Comment (PikalaxALT) [inline arm9/lib/src/MI_init.c:8]: Nani?!

Comment (red031000) [reply arm9/lib/src/MI_init.c:8]: not sure what's wrong with this? MI_WRAM_ARM7_ALL is the last value in the MIWram enum

Comment (PikalaxALT) [reply arm9/lib/src/MI_init.c:8]: I'm just surprised to see ARM7 in the ARM9 source, is all.

Comment (red031000) [reply arm9/lib/src/MI_init.c:8]: yeah it is kinda odd, but that's what it is according to the sdk /shrug

**Pull request merged**